CHANGES.rst

```diff
@@ -5,6 +5,7 @@ ChangeLog
 | Version    | Description                                                         | Date       |
 +============+=====================================================================+============+
 | *upcoming* | * Improve I2C render speed by not chunking into 32-byte blocks      | TBC        |
+|            | * Add support for bridging FTDI to I2C/SPI                          |            |
```

Comment:
I wouldn't mind calling this release 2.0

Reply:
Major version number change implies a breaking change though - which this is not.

Reply:
a breaking change or a big new feature (which this is), but yea, fine with me.